.github/workflows/robot/internal/bot/check.go

```diff
@@ -128,9 +133,20 @@ func (b *Bot) reviewersToDismiss(ctx context.Context) ([]string, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	internalApprovals := 0
 	reviewedBy := make(map[string]struct{})
 	for _, review := range reviews {
 		reviewedBy[review.Author] = struct{}{}
+		if review.State == "APPROVED" && b.c.Review.IsInternal(review.Author) {
```

Comment:
nit: replace the string literal with the `approved` const in `robot/internal/review`?

If we export those, same goes for the test cases in `check_test` I guess.

Reply:
Will do.